app/services/scrapers/TulsaCity/tulsa_inmates.rb

```diff
@@ -0,0 +1,43 @@
+require 'byebug'
+require 'json'
+module Scrapers
+  module TulsaCity
+    class TulsaInmates
+      attr_reader :inmates_json
+
+      def initialize
+        @inmates = []
+
+        @inmates_json = []
+      end
+
+      def self.perform
+        new.perform
+      end
+
+      def self.perform_inmates
+        new.perform_inmates
+      end
+
+      def perform
+        inmate_json = perform_inmates
+        bar = ProgressBar.new(inmate_json.count)
+        inmate_json.each do |inmate|
+          TulsaCityWorker
+            .set(queue: :high)
```

Comment:
not blocking but you don't need to set this queue as high here since it's already set line 7 of tulsa_city_worker.rb (you just explicitly set it if you want to use something other than the default)